docs/features/cameras.md

```diff
@@ -26,21 +26,25 @@ You can add up to four cameras to display on your dashboard.
 
 Currently supported types are;
 
-- MJPEG Stream
+- MJPEG Stream  
```

Comment:
```suggestion
- MJPEG Stream
```

Reply:
these changes are intentional, they improve the layout (making `MJPEG Stream` a "heading" in this case)
without the two spaces there wouldn't be a linebreak
i don't think it's worth opening another PR for these small changes

Reply:
Fair enough, that makes total sense! 🙂